### PR TITLE
DB2 is deprecated

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/refguide/general/system-requirements.md [hunk 1]
```diff
@@ -159,14 +159,14 @@ Mendix tries to support the most recent and patched database server versions fro
 
 Current support:
 
-* [IBM DB2](/refguide/db2/) 11.1 and 11.5 for Linux, Unix, and Windows
+* [Azure SQL](https://docs.microsoft.com/en-us/sql/t-sql/statements/alter-database-transact-sql-compatibility-level?view=sql-server-2017) v12 compatibility mode 140 or higher
```
Comment (basdebakker) [inline content/en/docs/refguide/general/system-requirements.md]: I realize you've moved it to make the order alphabetical. But Azure is basically SQL Server, which is why they were listed together.

Comment (MarkvanMents) [reply content/en/docs/refguide/general/system-requirements.md]: OK - I'll put it back. Thanks for the additional information.

Comment (MarkvanMents) [reply content/en/docs/refguide/general/system-requirements.md]: Done.

**Comment by MarkvanMents** [comment]
Approved over Slack.

**Pull request merged**